test/README.md

```diff
@@ -37,6 +37,7 @@ A utility is included to create test data to the cloudant database.
   ```
      python -m helpers.dataload -load 200
   ```
+- Run `python -m helpers.schema -addDocs` to add docs for schema test 
 
```

Comment:
Please add this additional comment:

_Note_: To load custom documents, put them in folder `test/resources/<database>.json` where <database> represents the database to load the document into. An invalid database name will be ignored. Documents are supplied in a `docs` array for bulk loading, e.g.:

`test/resources/n_customer.json`
```
{
    "docs":[
        {
            "_id": "helper.schema",
            "name":{
                "firstname":"Jim",
                "lastname":"Yal"
            },
            "age":20,
            "total_miles":true
        },
        {
            "name":1,
            "age":true
        },
        {
            "name":"Kaaa",
            "age":{"age":30}
        },
        {
            "_id": "z",
            "miles_ytd":true
        }
    ]
}
```